### PR TITLE
feat: home-page

### PR DESCRIPTION
**Pull request by dusariudaykumar**
# HOME PAGE
- added components for homepage

### CODE REVIEW

File: my-app/src/components/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import "../styles/homepage.css";
+const Footer = () => {
+  return (
+    <div>
+      <footer class="footer flex">
+        <div class="footer-left">
+          <h4>Quick Links</h4>
+          <ul class="footer-nav-links">
+            <li>
+              <a href="#">Home</a>
```
Comment (sakshi006) [inline my-app/src/components/Footer.jsx]: Can use 'Link' tags here

File: my-app/src/pages/HomePage.jsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      bannerIMG,
+      consolesIMG,
+      diskConsoleIMG,
+      categories: { preOrderBtn, allPS5GamesBtn, accessoriesBtn },
+    },
+  } = images;
+  return (
+    <div>
+      <main class="main-wrapper">
+        <div class="main-wrapper">
```
Comment (sakshi006) [inline my-app/src/pages/HomePage.jsx]: why two elements with same class name? any specific reason?

**Review by sakshi006 — APPROVED** [review]
Looks good to me.
Minor suggestion : since we are gonna code in react, you can make changes in 'li' tags, 'class' (make it className as per react)
Other than this, it is fine.
Naming convention actually conveys what a particular element is going to render.

File: my-app/src/components/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            </li>
+            <li>
+              <a href="#">Products</a>
+            </li>
+            <li>
+              <a href="#">My Cart</a>
```
Comment (Asuraking1n) [inline my-app/src/components/Footer.jsx]: Why using a tag ? 
You can use Link from react-router-dom
By using a tag your dom will reload again.

File: my-app/src/styles/footer.css [hunk 4]
```diff
@@ -0,0 +1,32 @@
+.footer {
+  width: 100%;
+  right: 0;
```
Comment (Asuraking1n) [inline my-app/src/styles/footer.css]: What's the point of right:0 without any position property ?

**Review by Asuraking1n — COMMENTED** [review]
Other thing looks good, 
Left some suggestions where u can correct in my opinion

**Pull request merged**